system-addon/lib/OnboardingMessageProvider.jsm

```diff
@@ -40,7 +42,8 @@ const ONBOARDING_MESSAGES = [
       button_label: "Try It Now",
       button_action: "OPEN_ABOUT_PAGE",
       button_action_params: "addons"
-    }
+    },
+    targeting: "isFirstRun && isInExperimentCohort == 1"
```

Comment:
Wouldn't this be `0` and `1` based on your default value of `0` above?

Reply:
i was thinking of having 0 be like no experiment running kind of thing, and then 1 or 2 based on the cohort. that way if for some reason the pref gets read wrong or something then it won't accidentally get put into a cohort. there just won't be any messages shown

Reply:
ah, that makes sense ok